core/tests/protractor_desktop/adminTabFeatures.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for collections.
+ */
+
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var AdminPage = require('../protractor_utils/AdminPage.js');
+
+describe('Admin Tab', function() {
+  var adminPage = null;
+
+  beforeAll(function() {
+    adminPage = new AdminPage.AdminPage();
+  });
+
+  it('assign and show roles correctly', function() {
```

Comment:
`should show and allow assigning role` (or something around this line)

Reply:
I would suggest having one test per role and also to have test for the "expected behaviour" of the assignment. Like if you assign someone collection editor role then check whether they can edit collection.)

I'm not sure whether we want to have such extensive tests considering e2e test takes lots of time, @nithusha21 any thoughts?

Reply:
I don't think so, these seem like redundant tests. acl_decorators_tests cover a variety of scenarios related to this. Here, let us just make sure that all the roles can be assigned and viewed (similar to what is done in the PR). Maybe, one improvement can be to have a list of possible roles, and then do this in a loop instead of repeating the same set of lines of code.